src/biogeophys/EnergyFluxType.F90

```diff
@@ -276,7 +280,7 @@ subroutine InitAllocate(this, bounds)
   end subroutine InitAllocate
     
   !------------------------------------------------------------------------
-  subroutine InitHistory(this, bounds, is_simple_buildtemp)
+  subroutine InitHistory(this, bounds, is_simple_buildtemp, is_prog_buildtemp)
```

Comment:
@olyson this isn't on you (really it's on me), but I'm wondering about sending both is_simple_buildtemp, and is_prog_buildtemp down to InitHistory. Really they exclusive they can't both be true at the same time (or both false at the same time). But, the way they are done you need to use both to really be sure of the behavior. So how you handled it is correct, but it's causing me to rethink how this interface works. I'm not quite sure I see what a better solution would be though. But, I think it's worth thinking about.

Reply:
I can see your point.  An alternative is to use "if ( .not. is_simple_buildtemp)" but then it's less obvious what's going on.  Not sure whether it's worth changing the behavior at this point.  Once we drop support for CLM4.5 it won't be used, although not sure when that will be.

Reply:
Noted this in issue #1653